flipper-sequel.gemspec

```diff
@@ -0,0 +1,26 @@
+# -*- encoding: utf-8 -*-
+require File.expand_path('../lib/flipper/version', __FILE__)
+
+flipper_sequel_files = lambda { |file| file =~ /sequel/ }
+
+Gem::Specification.new do |gem|
+  gem.authors       = ["John Nunemaker"]
+  gem.email         = ["nunemaker@gmail.com"]
+  gem.summary       = "Sequel adapter for Flipper"
+  gem.description   = "Sequel adapter for Flipper"
+  gem.license       = "MIT"
+  gem.homepage      = "https://github.com/jnunemaker/flipper"
+
+  extra_files = [
+    "lib/generators/flipper/templates/migration.rb",
```

Comment:
I believe this should be `lib/generators/flipper/templates/sequel_migration.rb`.